model/template.go

```diff
@@ -16,11 +16,11 @@ type Template struct {
 	Name           string `json:"name"`
 	IsSystem       string `json:"isSystem"`
 	Description    string `json:"description"`
-	DefaultVersion string `json:"defaultVersion" yaml:"version"`
+	DefaultVersion string `json:"defaultVersion" yaml:"default_version"`
```

Comment:
I guess we decided yes to this at some point since it's in rancher/rancher#8319.